app/dashboards/user_dashboard.rb

```diff
@@ -65,7 +72,7 @@ class UserDashboard < Administrate::BaseDashboard
   FORM_ATTRIBUTES = %i[
     name
     email
-    password
```

Comment:
This field is required to add an AdminUser from Administrate. Please don't remove it.

Reply:
Do some tests. If this field is added, admin can't change any field (for example: avatar or email). I recommend to create AdminUser by changing Regular User to Admin User (We must add field "type") - it's the simplest way.

Reply:
OK, we will discuss about this topic on Hangout.
Now leave the password removed.